src/Http/Controllers/Support/ResolveController.php

```diff
@@ -122,4 +123,33 @@ public function resolveIdsToNames(Request $request)
 
         return response()->json($response);
     }
+
+    public function resolveMainCharacter(Request $request)
+    {
+
+        // Init the initial return array
+        $response = collect();
+
+        // Grab the ids from the request for processing
+        collect(explode(',', $request->ids))->map(function ($id) {
+
+            // Convert them all to integers
+            return (int) $id;
+
+        })->each(function ($chunk) use (&$response) {
+
+            $character = User::find($chunk);
+
+            if (is_null($character)) {
+                $maincharacter = null;
+            } else {
+                $maincharacter = $character->group->main_character;
```

Comment:
it can be switch to a ternary `$main_character = is_null($character) ? null : $character->group->main_character;`